wazo_acceptance/prerequisite.py

```diff
@@ -266,6 +269,25 @@ def _configure_consul(context):
         context.remote_sysutils.restart_service('consul')
 
 
+def _configure_nat(context):
+    if not context.wazo_config['prerequisites']['nat']['enabled']:
+        logger.debug('Skipping NAT configuration')
+
+    transport_udp = context.helpers.sip_transport.get_by(name='transport-udp')
+    external_ip = context.wazo_config['wazo_host']
```

Comment:
As discussed, we need an IP address here, but `wazo_host` may be a hostname, not an IP address.

Reply:
ye